tests/test_uefi.py

```diff
@@ -102,7 +102,7 @@ def my_onexit(ql, address, params):
             ql.run()
 
             self.assertEqual(0, self.set_api)
-            self.assertEqual(21475885153, self.set_api_onenter)
+            self.assertEqual(2014052449, self.set_api_onenter)  # magic value was 21475885153 (?)
```

Comment:
we need @lib2k and @assafcarlsbad to confirm the value.

Reply:
My guess is that the value was derived from the original stack / heap locations that were changed. But I couldn't really find out..

Reply:
No clue, I didn't write this file. Looks like it was added in this commit: https://github.com/qilingframework/qiling/commit/a4e560ef75a7fddd1a1d6ce93212a6e8451c2fef